quesma/quesma/config/config_v2.go

```diff
@@ -552,6 +552,9 @@ func (c *QuesmaNewConfiguration) TranslateToLegacyConfig() QuesmaConfiguration {
 					errAcc = multierror.Append(errAcc, fmt.Errorf("invalid target %s in configuration of %s", target, DefaultWildcardIndexName))
 				}
 			}
+			if defaultConfig.UseCommonTable {
+				conf.CreateCommonTableForWildcard = true
```

Comment:
Shouldn't this also set both `conf.CreateCommonTable = true` and `conf.CreateCommonTableForWildcard = true`? (the first one determines if common table is created, the second one controls `table_resolver`)

Reply:
ok, that make sense

Reply:
However, after consideration I don't think it's needed as there are two separate rules:
- makeDefaultWildcard
- makeCommonTableResolver

Reply:
What I had in mind was this use of `CreateCommonTable`: https://github.com/QuesmaOrg/quesma/blob/87c168d36135bb99b47c0b77ec1e089aa8f65cb5/quesma/main.go#L105-L108

Reply:
ok, then you are right